www/source/docs/reference/reference.html.md.erb

```diff
@@ -1260,13 +1260,43 @@ Packages are installed in the `/hab/pkgs/` directory, and then further organized
 ---
 ## <a name="sup-log-configuration" id="sup-log-configuration" data-magellan-target="sup-log-configuration">Supervisor Log Configuration Reference</a>
 
-With the 0.83.0 release of the Chef Habitat Supervisor, it is possible to have greater control over logging output, including the ability to dynamically adjust the logging configuration of a running Supervisor. There are two main ways of configuring logging, each of which has their own strengths and weaknesses: using environment variables, and using a configuration file.
+With the 0.83.0 release of the Chef Habitat Supervisor, it is possible to have greater control over logging output, including the ability to dynamically adjust the logging configuration of a running Supervisor. There are three ways of configuring logging, each of which has their own strengths and weaknesses: as part of the `hab` command, by using environment variables, and by using a configuration file.
 
-### Environment Variable Configuration
+### RUST_LOG with a command
+
+Use `RUST_LOG=debug RUST_BACKTRACE=1` to see an individual command's debug and backtrace.
+
+    ```bash
+    # Linux/MacOS
+    # replace "hab sup run" with your command
+    RUST_LOG=debug RUST_BACKTRACE=1 hab sup run
+    ```
+
+    ```powershell
+    # Windows
+    # replace "hab sup run" with your command
+    $env:RUST_LOG="debug"; $env:RUST_BACKTRACE=1; hab sup run
+    ```
+
+### RUST_LOG as an Environment Variable Configuration
 
 It is still possible to configure logging via the `RUST_LOG` environment variable. This approach is often useful for quickly re-configuring logging (requiring a restart), or for easily configuring logging in container-based deployments.
 
-The configuration scheme is essentially the same as that described in the documentation of the Rust [env_logger](https://docs.rs/env_logger/0.6.1/env_logger/#enabling-logging) crate, with the exception that we do not allow additional regular expression-based filtering. The configuration values the Supervisor recognizes are described below:
+```bash
+export HAB_FEAT_EVENT_STREAM = “1”
+export RUST_LOG=“debug”
+export RUST_BACKTRACE=1
+```
+
+```powershell
+$env:HAB_FEAT_EVENT_STREAM = “1”
+$env:RUST_LOG=“debug”
+$env:RUST_BACKTRACE=1
+```
+
+### RUST_LOG in a Configuration File
```

Comment:
This reorganization has mixed up the original documentation.  `RUST_LOG` is never set in a configuration file; it is _always_ an environment variable. 

The original documentation laid out how to configure logging using either `RUST_LOG` _or_ the newer YAML configuration file. Internally, the value of `RUST_LOG` is essentially translated into what you could write in the config file, and is provided as a way to maintain backwards compatibility with the logging library we used before.

What follows here is explaining how you should structure the _content_ of the `RUST_LOG` variable. Below that is the explanation on how you should structure the content of the configuration file, should you choose that route.